### PR TITLE
[AB#47358] Updated the "Generate the workflows package" section in the readme related to a support ticket

### PR DESCRIPTION
**Pull request by Nimasha-axinom**
# Pull Request

## Prerequisites

- [ ] The PR is targeting the right branch (`dev` for features and `master` for
      releases)
- [ ] potential **release notes** to the PR description added
- [ ] potential **testing notes** to the PR description added
- [ ] appropriate labels for the PR applied

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -197,10 +197,17 @@ yarn run pilet publish --no-fresh --url "https://frontends.service.eu.axinom.net
...
+[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) 
+that has the `Pilets: Publish` permissions under the "Micro Frontend Service" as `api-key`. This
 value will be shown to you alongside with the other secrets when creating a
 service account or when generating new secrets for an existing account.
 
+For authentication, you need a Client ID and Client Secret of this service account. 
+
+If you have both values, the API key is just base64($clientid:$clientSecret). For 
+base64 you can use a tool in our Portal: https://portal.axinom.com/mosaic/tools/base64 
+(or any other base64 tool).
```
Comment (EmperorRXF) [inline README.md]: something like this makes sense?

-----

If you already have the plaintext `Client ID` & the `Client Secret` of the service account, you can generate the `base64($clientid:$clientSecret)` to be used as the `api-key` using a tool BASE64 generator tool, such as: https://portal.axinom.com/mosaic/tools/base64

File: README.md [hunk 2]
```diff
@@ -197,10 +197,17 @@ yarn run pilet publish --no-fresh --url "https://frontends.service.eu.axinom.net
 ```
 
 Please note, that you need to provide the 'Base64 Encoded Credentials' of a
-service account that has the `Pilets: Publish` permissions as `api-key`. This
+[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) 
+that has the `Pilets: Publish` permissions under the "Micro Frontend Service" as `api-key`. This
```
Comment (EmperorRXF) [inline README.md]: slightly change to something like this?

-----

Please note, that you need to provide the _Base64 Encoded Credentials_ of a
[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) as the value of `api-key`. This service account must contain the `Pilets: Publish` permission of the "Micro Frontend Service" in order to be authorized to perform the action of publishing the workflows.

**Review by EmperorRXF — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -197,10 +197,17 @@ yarn run pilet publish --no-fresh --url "https://frontends.service.eu.axinom.net
...
 value will be shown to you alongside with the other secrets when creating a
 service account or when generating new secrets for an existing account.
 
+For authentication, you need a Client ID and Client Secret of this service account.
```
Comment (BenjaminSchwendner) [inline README.md]: this should either be removed, or moved further up.
As this is the general info, that the call needs the service account details - and then we can explain which permissions it needs and how to obtain it:
1. on creation we show the calculate value
2. if you have the plaintext values already, you can use the base64 tool, etc

File: README.md (hunk 1 above)
Comment (Nimasha-axinom) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (Nimasha-axinom) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (Nimasha-axinom) [reply README.md]: Fixed

File: README.md [hunk 4]
```diff
@@ -195,11 +195,21 @@ To deploy the workflows into an environment run a command like:
...
 
 Please note, that you need to provide the 'Base64 Encoded Credentials' of a
-service account that has the `Pilets: Publish` permissions as `api-key`. This
-value will be shown to you alongside with the other secrets when creating a
-service account or when generating new secrets for an existing account.
+[service account](https://docs.axinom.com/platform/core/identity/authenticate-serviceaccount/#what-is-a-service-account) 
+as the value of `api-key`. 
+
+If you have the plaintext Client ID & the Client Secret of the service account, 
+you can generate the base64($clientid:$clientSecret) to be used as the `api-key`
```
Comment (BenjaminSchwendner) [inline README.md]:
```suggestion
you can generate the `base64($clientid:$clientSecret)` to be used as the `api-key` 
```

**Review by BenjaminSchwendner — APPROVED** [review]

**Pull request merged**